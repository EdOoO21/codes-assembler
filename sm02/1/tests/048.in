-354 -730 140 457 95 -381 624 921 34 -606 462 560 -108 -661 -684 978 652 789 346 -780 9 329 -549 -554 561 47 139 741 41 -683 -34 -556 -943 -850 -133 -870 -729 21 376 -667 946 -737 -885 429 122 -403 917 -306 430 -309 -806 723 -512 -444 -458 327 -971 916 805 169 840 -421 292 -717 -279 336 -105 -641 -591 -178
