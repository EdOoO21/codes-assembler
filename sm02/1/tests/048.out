-178
-591
-641
-105
336
-279
-717
292
-421
840
169
805
916
-971
327
-458
-444
-512
723
-806
-309
430
-306
917
-403
122
429
-885
-737
946
-667
376
21
-729
-870
-133
-850
-943
-556
-34
-683
41
741
139
47
561
-554
-549
329
9
-780
346
789
652
978
-684
-661
-108
560
462
-606
34
921
624
-381
95
457
140
-730
-354
